### DISCUSSION TITLE
Add ability to store multiple App Store Connect API keys via match (or similar tool).

### DISCUSSION THREAD

**Discussion by Katsz** [fastlane/fastlane#18215] [Ideas]
`Match` [was designed](https://docs.fastlane.tools/actions/match/#git-storage_1) to be able to store certs/provisions of multiple team via multiple branches. Nowadays with introduction of mandatory 2fa most developers will have to switch to App Store Connect API `.p8` key which more or less get supported in `fastlane`. However, when working on multiple teams, managing multiple keys is quite a hassle.

I think `fastlane` needs to support API keys management of some sort. `match` is already capable of encrypting certs using OpenSSL, so secure storage is out of the question. Maybe `match` isn't quite the place to store them, but encrypted repo idea may be reused in brand new `action`?

I want to hear what you guys think.

**Comment by dinsen** [comment]
Hi @Katsz 
There is already a discussion here https://github.com/fastlane/fastlane/discussions/18093
